meta-sokol-flex-distro/recipes-core/kbd/kbd_2.4.0.bbappend

```diff
@@ -1,13 +0,0 @@
-# ---------------------------------------------------------------------------------------------------------------------
-# SPDX-License-Identifier: MIT
-# ---------------------------------------------------------------------------------------------------------------------
-
-SRC_URI:prepend:sokol-flex = "git://github.com/MentorEmbedded/kbd;branch=2.4;protocol=https "
-SRC_URI:remove:sokol-flex := "${KERNELORG_MIRROR}/linux/utils/${BPN}/${BP}.tar.xz"
-SRCREV:sokol-flex = "a95b34e0f4c78be7e3c137613b3d8c161ab322ba"
-S:sokol-flex = "${WORKDIR}/git"
-
-DEPENDS:append:sokol-flex = " bison-native"
-
-# Don't exclude autopoint
-EXTRA_AUTORECONF:sokol-flex = ""
```

Comment:
Aren't the DEPENDS and EXTRA_AUTORECONF changes still needed even with the upstream sources? Also, why do we have the DEPENDS change here and not in staging?

@kergoth thoughts?

Reply:
I am able to buld successfully 
ahsann@pkl-melbuild-vm1:~/mel/releases/builds/ginkgo/async1/build_imx8mplus$ bitbake kbd
NOTE: Started PRServer with DBfile: /home/ahsann/mel/releases/builds/ginkgo/async1/build_imx8mplus/cache/prserv.sqlite3, Address: 127.0.0.1:45471, PID: 16136
WARNING: Low memory per CPU core (~1 Gb) in this system. You may wish to reduce BB_NUMBER_THREADS to 8 or less and reduce PARALLEL_MAKE to 8 or less in local.conf, or set MEM_PER_CPU_MESSAG
E="0" to disable this message.
WARNING: Host distribution "ubuntu-18.04" has not been validated with this version of the build system; you may possibly experience unexpected failures. It is recommended that you use a tes
ted distribution.
Loading cache: 100% |                                                                                                                                                        | ETA:  --:--:--
Loaded 0 entries from dependency cache.
Parsing recipes: 100% |#######################################################################################################################################################| Time: 0:00:28
Parsing of 2923 .bb files complete (0 cached, 2923 parsed). 4388 targets, 485 skipped, 5 masked, 0 errors.
NOTE: Resolving any missing task queue dependencies

Build Configuration:
BB_VERSION           = "2.0.0"
BUILD_SYS            = "x86_64-linux"
NATIVELSBSTRING      = "ubuntu-18.04"
TARGET_SYS           = "aarch64-oe-linux"
MACHINE              = "imx8mpevk-flex"
DISTRO               = "sokol-flex"
DISTRO_VERSION       = "13"
TUNE_FEATURES        = "aarch64 armv8a crc"
TARGET_FPU           = ""
EXTERNAL_TOOLCHAIN   = "/home/ahsann/siemens/embedded_arm_13.0.0/toolchains/aarch64-oe-linux.13.0"
EXTERNAL_TARGET_SYS  = "aarch64-oe-linux"
GCC_VERSION          = "11.3.0"
SOURCERY_TOOLCHAIN_VERSION = "13.0.1"
meta-flex-bsp-imx8mpevk
meta-flex-imx-common
meta-flex-bsp-common = "HEAD:aed5994d9603602a6421534356188da9a9ea83a2"
meta-sokol-flex-distro = "kirkstone:3fbf1c42c6613b598af13fd2dbe3b895ed38b780"
meta-flex-user-examples = "HEAD:8976af610b338d002a59ffdd98f6281f56f4f2d0"
meta-flex-private    = "HEAD:b55bcf73e1d27eee4bbf7acf030a6274b1cdb3d6"
meta-flex-cve        = "HEAD:5e9699328d71bae5435b59735294f6816e5d6af1"
meta-sokol-flex-staging = "kirkstone:3fbf1c42c6613b598af13fd2dbe3b895ed38b780"
meta-flex-swupdate   = "HEAD:aef89a81b3f8bffc3d353d2e9cc2570cfc1d76d7"
meta-v2x
meta-ml
meta-sdk
meta-bsp             = "HEAD:a806f71b72b878c505205d9195ea6237bf89c335"
meta-swupdate        = "HEAD:f27a63f42d124676a07401462b1c4a64f38b768b"
meta-networking
meta-multimedia
meta-python          = "HEAD:278ec081a64e6a7679d6def550101158126cd935"
meta-freescale       = "HEAD:c82d4634e7aba8bc0de73ce1dfc997b630051571"
meta-filesystems     = "HEAD:278ec081a64e6a7679d6def550101158126cd935"
meta                 = "HEAD:a8c82902384f7430519a31732a4bb631f21693ac"
meta-sokol-flex-support = "kirkstone:3fbf1c42c6613b598af13fd2dbe3b895ed38b780"
meta-freescale-distro = "HEAD:d5bbb487b2816dfc74984a78b67f7361ce404253"
meta-freescale-3rdparty = "HEAD:5977197340c7a7db17fe3e02a4e014ad997565ae"
meta-sourcery        = "HEAD:36ea81f7a016a658753ec2ffc731cd7274337bef"
meta-sokol-flex-common = "kirkstone:3fbf1c42c6613b598af13fd2dbe3b895ed38b780"
meta-oe              = "HEAD:278ec081a64e6a7679d6def550101158126cd935"
meta-external-toolchain = "HEAD:92449719aed57bdd841ce001861306e17d80053a"

Initialising tasks: 100% |####################################################################################################################################################| Time: 0:00:02
Checking sstate mirror object availability: 100% |############################################################################################################################| Time: 0:00:00
Sstate summary: Wanted 202 Local 0 Mirrors 0 Missed 202 Current 0 (0% match, 0% complete)
NOTE: Executing Tasks
NOTE: Tasks Summary: Attempted 818 tasks of which 0 didn't need to be rerun and all succeeded.
NOTE: Writing buildhistory
NOTE: Writing buildhistory took: 8 seconds

Summary: There were 2 WARNING messages.